Doctrine/Phpcr/SitemapDocumentProvider.php

```diff
@@ -42,7 +42,7 @@ public function load($sitemap)
     {
         // todo rewrite query when https://github.com/symfony-cmf/CoreBundle/issues/126 is ready
         $documentsCollection = $this->manager->createQuery(
-            'SELECT * FROM [nt:unstructured] WHERE (visible_for_sitemap = true)',
+            'SELECT * FROM [nt:unstructured] WHERE (visible_for_sitemap = true) ORDER By id',
```

Comment:
is there a property `id`? usually there is not, and the node path is mapped onto a class field $id. but you can't sort by the path afaik. did you verify that this works? in the search result, the field is called [jcr:path] but i doubt that that would work either.